plugins/available/less-pretty-cat.plugin.bash

```diff
@@ -1,31 +1,31 @@
+# shellcheck shell=bash
 cite about-plugin
 about-plugin 'pygmentize instead of cat to terminal if possible'
 
-if $(command -v pygmentize &> /dev/null) ; then
-  # get the full paths to binaries
-  CAT_BIN=$(which cat)
-  LESS_BIN=$(which less)
-  BASH_IT_CCAT_STYLE="${BASH_IT_CCAT_STYLE:=default}"
-  BASH_IT_CLESS_STYLE="${BASH_IT_CLESS_STYLE:=default}"
+_command_exists pygmentize || return
 
-  # pigmentize cat and less outputs - call them ccat and cless to avoid that
-  # especially cat'ed output in scripts gets mangled with pygemtized meta characters
-  function ccat()
-  {
-      about 'runs either pygmentize or cat on each file passed in'
-      param '*: files to concatenate (as normally passed to cat)'
-      example 'cat mysite/manage.py dir/text-file.txt'
-      for var;
-      do
-          pygmentize -f 256 -O style="$BASH_IT_CCAT_STYLE" -g "$var" 2>/dev/null || "$CAT_BIN" "$var";
-      done
-  }
+# pigmentize cat and less outputs - call them ccat and cless to avoid that
+# especially cat'ed output in scripts gets mangled with pygemtized meta characters
+function ccat() {
+	about 'runs either pygmentize or cat on each file passed in'
+	param '*: files to concatenate (as normally passed to cat)'
```

Comment:
I couldn't find any other use of `*` in params within bash-it and my search-fu failed me - I don't have a specific issue with it, but wanted to confirm it was standard, but could not confirm ...

Reply:
I have no idea

Reply:
Yea me either ... I'm okay with it, we'll have to see how the generated docs look and if there are any issues with them.

Assuming not, I say its now our standard for all-arg parms :)